packages/falso/src/tests/random-uuid.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { randomUuid } from '../lib/random-uuid';
+describe('RandomUUIDTest', () => {
+  it('Should match UUID regex', () => {
+    const uuid = randomUuid();
+    const regex =
+      /^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$/g;
+    expect(uuid.match(regex)).toBeTruthy();
+  });
+
+  it("Shouldn't duplicate element into a range of 10", () => {
```

Comment:
Sorry to be a pain but for consistency, I think this should use single quotes. (With a backslash to escape the apostrophe)

Reply:
Hi @theryansmee, I agree and that's what I did, but Prettier turned it into double quotes. I will consider writing a positive sentence.

Reply:
Ah sorry, I don't had a lot of experience with prettier. Maybe `should not` to fudge it? ha

Reply:
@matheusnascgomes Ah sorry, I don't had a lot of experience with prettier. Maybe `should not` to fudge it? ha

Reply:
With lint-staged on the project and given the prettier configuration used, the file should be prettified with single quotes.
As I have an upcoming PR for the project, I'll check if it works as expected.

Reply:
I just checked and my staged spec file was prettified from double quotes to single quotes as expected.

You must have a local setting overriding the project's one.

Reply:
For strings, Prettier will prefer the configured form, but will change to the othe when the string contains apostrophes or quotes.
For consistency, it's better to just use back ticks. Prettier won't change those